i18n/states.php

```diff
@@ -172,6 +172,20 @@
 	),
 	'BH' => array(),
 	'BI' => array(),
+	'BJ' => array( // Benin states.
+		'BJ-01' => __( 'Alibori', 'woocommerce' ),
+		'BJ-02' => __( 'Atacora', 'woocommerce' ),
```

Comment:
```suggestion
		'AK' => __( 'Atakora', 'woocommerce' ),
```

I think we should keep what the CLDR uses, it can be translated in other languages too.